### ISSUE TITLE
Mortars

### ISSUE THREAD

**Issue by stk2008** [CleverRaven/Cataclysm-DDA#15284]
Was thinking why we dont have Mortars in the game.

Could be fun to set up a mortar not sure the best way to have you sight it in though.

would be awesome if we could aim outside of the current player bubble and hit targets many miles away.

I understand there is lots wrong with this.

are enemies even on the map outside of player bubble?
how can we handle aiming?

I shall dig some info up on mortars and then post it here for Ref.

**Comment by Coolthulhu** [comment]
> we could aim outside of the current player bubble

Very unlikely to happen. Not just for one small thing like mortars and certainly not until critters other than zombies can move on the overmap.
> are enemies even on the map outside of player bubble?

No.
> how can we handle aiming?

Not with the current system - it would result in giant inaccuracy at that range.

The closest we could get to mortars would be parabolic projectiles, but within reality bubble. For example, throwing a grenade over a wall.

**Comment by kevingranade** [comment]
Grenades within the reality bubble would be cool, including grenade
launchers.  We need more complete 3D info before that makes sense.
If you were aiming outside the bubble, you'd be aiming at a map tile
instead of a square, this would rely on blasts to do damage rather than
direct hits. The tricky part there is we'd need to load the area the
explosive went off in on demans, apply the explosion, and save it. That's
not too hard, just a new chunk of map handling code.

**Comment by elricsfate** [comment]
Implementation of this would be pretty complex.

Aiming could require the player to utilize either an artillery chart (Firing table) for the piece in question or an artillery computer. 

Providing "real life functionality" where one can adjust the elevation of the weapon might not be plausible. 

Instead allowing the player to simply mark a portion of the map and have the round fired in that direction to the best of the characters ability would probably be the easiest way. Firing tables could be less accurate than the artillery computer.

**Comment by kevingranade** [comment]
Yes, I was definitely thinking just marking a target on the map.

**Comment by illi-kun** [comment]
### This issue was closed as it appears inactive.

Reducing open issues to those which are (or will) be actively worked upon helps us focus our efforts. This issue has not been deleted - it still appears in searches and if it contains relevant information you are encouraged to continue to link to it.

### If this issue was a bug
It should be reopened if it can be reproduced in the current build. You can obtain the most recent copy [here](http://dev.narc.ro/cataclysm/jenkins-latest/). Please check there is not a more recent report of this bug before doing so. If no more recent report exists you should continue the discussion in this issue.

### If this was a feature request
If the consensus was that the idea was good you could consider submitting an implementation via a PR. If you want to comment further please do so here as opposed to opening a new issue. Before posting check nobody has already made the same point and consider whether your comments are likely to lead to an implementation. If you have doubts about either consider instead [voting](http://iv.cataclysmdda.com:8111/all) for the issue

### If you want to work on this issue
Then either assign it to yourself or if you are unable to do so claim it via adding a comment. Please don't assign others or make a general request for action.
